File: Programming Basics/Conditional Statements - Exercises/07.Shopping.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        double budget = double.Parse(Console.ReadLine());
        int videocards = int.Parse(Console.ReadLine());
        int processors = int.Parse(Console.ReadLine());
        int ram = int.Parse(Console.ReadLine());

        double videocardsPrice = videocards * 250;
        double processorsPrice = (videocardsPrice * 0.35) * processors;
        double ramPrice = (videocardsPrice * 0.1) * ram;

        double totalPrice = videocardsPrice + processorsPrice + ramPrice;

        if (videocards > processors)
        {
            totalPrice = totalPrice - (totalPrice * 0.15);
        }

        if (budget >= totalPrice)
        {
            Console.WriteLine($"You have {budget - totalPrice:f2} leva left!");
        }
        else
        {
            Console.WriteLine($"Not enough money! You need {totalPrice - budget:f2} leva more!");
        }
    }
}

File: Programming Basics - Python/Conditional Statements - Exercises/07.Shopping.py
budget = float(input())
gpus = int(input())
cpus = int(input())
rams = int(input())

gpus_cost = 250 * gpus
cpus_cost = (gpus_cost * 0.35) * cpus
rams_cost = (gpus_cost * 0.10) * rams

total_cost = gpus_cost + cpus_cost + rams_cost

if gpus > cpus:
    total_cost -= total_cost * 0.15

if budget >= total_cost:
    print(f'You have {budget - total_cost:.2f} leva left!')

else:
    print(f'Not enough money! You need {total_cost - budget:.2f} leva more!')